spec.html

```diff
@@ -35671,6 +36081,94 @@ <h1>Runtime Semantics: CompileToCharSet</h1>
+        <emu-grammar>ClassIntersection :: ClassOperand `&amp;&amp;` ClassOperand</emu-grammar>
+        <emu-alg>
+          1. Let _A_ be CompileToCharSet of the first |ClassOperand|.
+          1. Let _B_ be CompileToCharSet of the second |ClassOperand|.
+          1. Return the intersection of CharSets _A_ and _B_.
+        </emu-alg>
+        <emu-grammar>ClassIntersection :: ClassIntersection `&amp;&amp;` ClassOperand</emu-grammar>
+        <emu-alg>
+          1. Let _A_ be CompileToCharSet of the |ClassIntersection|.
+          1. Let _B_ be CompileToCharSet of the |ClassOperand|.
+          1. Return the intersection of CharSets _A_ and _B_.
+        </emu-alg>
+
+        <!-- ClassSubtraction -->
+        <emu-grammar>ClassSubtraction :: ClassOperand `--` ClassOperand</emu-grammar>
+        <emu-alg>
+          1. Let _A_ be CompileToCharSet of the first |ClassOperand|.
+          1. Let _B_ be CompileToCharSet of the second |ClassOperand|.
+          1. Return the subtraction of CharSets _A_ and _B_.
+        </emu-alg>
+        <emu-grammar>ClassSubtraction :: ClassSubtraction `--` ClassOperand</emu-grammar>
+        <emu-alg>
+          1. Let _A_ be CompileToCharSet of the |ClassSubtraction|.
+          1. Let _B_ be CompileToCharSet of the |ClassOperand|.
+          1. Return the subtraction of CharSets _A_ and _B_.
+        </emu-alg>
+
+        <!-- ClassOperand -->
+        <emu-grammar>ClassOperand :: ClassCharacter</emu-grammar>
+        <emu-alg>
+          1. Let _A_ be CompileToCharSet of |ClassCharacter|.
+          1. Return ! MaybeSimpleCaseFolding(_A_).
+        </emu-alg>
+        <emu-grammar>ClassOperand :: ClassStrings</emu-grammar>
+        <emu-alg>
+          1. Let _A_ be CompileToCharSet of |ClassStrings|.
+          1. Return ! MaybeSimpleCaseFolding(_A_).
+        </emu-alg>
+        <emu-grammar>ClassOperand :: NestedClass</emu-grammar>
+        <emu-alg>
+          1. Return CompileToCharSet of |NestedClass|.
+        </emu-alg>
+
+        <!-- ClassOperand -->
+        <emu-grammar>NestedClass :: `[` ClassRanges `]`</emu-grammar>
+        <emu-alg>
+          1. Return CompileToCharSet of |ClassRanges|.
+        </emu-alg>
+        <emu-grammar>NestedClass :: `[` `^` ClassRanges `]`</emu-grammar>
+        <emu-alg>
+          1. Let _A_ be CompileToCharSet of |ClassRanges|.
+          1. Return the CharSet containing all characters not in _A_.
+        </emu-alg>
+        <emu-grammar>NestedClass :: `\` CharacterClassEscape</emu-grammar>
+        <emu-alg>
+          1. Return CompileToCharSet of |CharacterClassEscape|.
+        </emu-alg>
+
+        <!-- ClassRange -->
+        <emu-grammar>ClassRange :: ClassCharacter `-` ClassCharacter</emu-grammar>
+        <emu-alg>
+          1. Let _A_ be CompileToCharSet of the first |ClassCharacter|.
+          1. Let _B_ be CompileToCharSet of the second |ClassCharacter|.
+          1. Return ! MaybeSimpleCaseFolding(! CharacterRange(_A, B_)).
```

Comment:
```suggestion
          1. Return ! MaybeSimpleCaseFolding(! CharacterRange(_A_, _B_)).
```
(@bakkot already made this suggestion, but it got unmoored during the rebase, I think.)